new(sinsp): Phase 1 / 3 for capture-guided loadgen

nathan-b opened the pull request:
The capture-guided load generator is a new tool based on libsinsp. The goal is to use a capture file to guide load generation to create a more organic, real-world load pattern even as the load itself is synthetic.

Phase 1: Read and classify the capture file
Phase 2: Build the list of events to generate
Phase 3: Generate the events list

This commit has some simple output to validate that the functionality is working correctly. This output will not be present in the final version once all three phases are implemented.

/kind feature
/area libscap

```release-note
NONE
```

Commit:
new(sinsp): Phase 1 / 3 for capture-guided loadgen

The capture-guided load generator is a new tool based on libsinsp. The
goal is to use a capture file to guide load generation to create a more
organic, real-world load pattern even as the load itself is synthetic.

Phase 1: Read and classify the capture file
Phase 2: Build the list of events to generate
Phase 3: Generate the events list

This commit has some simple output to validate that the functionality is
working correctly. This output will not be present in the final version
once all three phases are implemented.

Signed-off-by: Nathan Baker <nathan.baker@sysdig.com>

pgcrooks-sysdig reviewed (approved):
I like where this is going :-)

Commit:
new(sinsp): Phase 2 / 3 for capture-guided loadgen

    The capture-guided load generator is a new tool based on libsinsp. The
    goal is to use a capture file to guide load generation to create a more
    organic, real-world load pattern even as the load itself is synthetic.

    Phase 1: Read and classify the capture file
    Phase 2: Build the list of events to generate <== We are here
    Phase 3: Generate the events list

    This commit has some simple output to validate that the functionality is
    working correctly. This output will not be present in the final version
    once all three phases are implemented.

    This commit is also not complete, but it should be documented where
    more work is needed

    Signed-off-by: Nathan Baker <nathan.baker@sysdig.com>
